packages/netlify/src/ssr-function.ts

```diff
@@ -27,13 +27,7 @@ export const createExports = (manifest: SSRManifest, _args: Args) => {
 
 			locals.netlify = { context };
 
-			const response = await app.render(request, { routeData, locals });
-
-			if (app.setCookieHeaders) {
-				for (const setCookieHeader of app.setCookieHeaders(response)) {
-					response.headers.append('Set-Cookie', setCookieHeader);
-				}
-			}
+			const response = await app.render(request, { routeData, locals, addCookieHeader: true, clientAddress: context.ip });
```

Comment:
nit: This change seems unrelated to this PR. I would personally prefer to separate it.

Reply:
oof, that commit was still in my working copy and I didn't notice it! will remove it.

Reply:
@Skn0tt do you still want to remove the commit? We can also keep it in the PR if you want.

Reply:
removed it!